docker/build-image.sh

```diff
@@ -98,25 +98,25 @@ TAGS=()
 
 cd ..
 
-# If neither --focal nor --jammy is specified, build only focal for now.
-# Once all humble related changes are merged into develop, jammy should be enabled.
+# If neither --noetic-foxy nor --humble is specified, build only noetic-foxy for now.
+# Once all humble related changes are merged into develop, humble should be enabled.
 # https://usdot-carma.atlassian.net/browse/ARC-227
-if [ "$BUILD_FOCAL" = false ] && [ "$BUILD_JAMMY" = false ]; then
-    BUILD_FOCAL=true
-    BUILD_JAMMY=false
+if [ "$BUILD_NOETIC_FOXY" = false ] && [ "$BUILD_HUMBLE" = false ]; then
+    BUILD_NOETIC_FOXY=true
+    BUILD_HUMBLE=false
 fi
 
 # TODO, distinguish with suffix when Humble is fully integrated
-# until then focal will have no suffix and be the main image
+# until then noetic-foxy will have no suffix and be the main image
 # https://usdot-carma.atlassian.net/browse/ARC-227
-if [ "$BUILD_FOCAL" = true ]; then
-    echo "Building carma-base focal image"
-    build_image "focal/Dockerfile" "" #replace with "-focal"
+if [ "$BUILD_NOETIC_FOXY" = true ]; then
+    echo "Building carma-base noetic-foxy image"
+    build_image "noetic-foxy/Dockerfile" "" #replace with just "-noetic"
```

Comment:
Is this replace comment supposed to be after the migration to humble is complete? I think its not a required comment, but if we are adding it, we should probably update it to something like "replace with just "-noetic" after ros2 is migrated to humble"

Reply:
Specified to your suggestion.